x/authenticator/authenticator/interface.go

```diff
@@ -5,46 +5,68 @@ import (
 )
 
 const (
+	// SignatureVerificationAuthenticator is a Type of authenticator that specific deals with
+	// secp256k1 signatures,
```

Comment:
this also supports secp256r1 and multisig, right?

Reply:
It supports multisig, but not `secp256r1`! I'm currently writing tests for multisig :muscle: 

`secp256r1` I think should be it's own authenticator, as we would like users to opt in to using this.

Reply:
ahh, I thought it did because the DefaultSigVerificationGasConsumer has it. But yeah, It prob should be its own